### DISCUSSION TITLE
How should we cite pixi?

### DISCUSSION THREAD

**Discussion by twrightsman** [prefix-dev/pixi#881] [Q&A] [answered]
Is there a preferred way of citing pixi? I currently have the following in BibTeX, using guidance from the [Academia StackExchange](https://academia.stackexchange.com/questions/14010/how-do-you-cite-a-github-repository):

```bibtex
@misc{bib-pixi,
  author = {Arts, Ruben and Zalmstra, Bas and de Jager, Tim and Vollprecht, Wolf},
  title = {pixi},
  year = {2023},
  publisher = {GitHub},
  journal = {GitHub repository},
  howpublished = {\url{https://github.com/prefix-dev/pixi}},
  commit = {745862373f60a90c77b70fb68365af54614d571e}
}
```

This renders in APA format as:

> Arts, R., Zalmstra, B., Jager, T. de, & Vollprecht, W. (2023). pixi. GitHub.

Is this appropriate? I have little idea which contributors should be listed and in which order, so I restricted it to just the prefix.dev [team](https://prefix.dev/team).

**Comment by pavelzw** [comment]
GitHub supports citation files: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files

**Reply by twrightsman** [reply]
Good point @pavelzw; Pixi is lacking one at the moment.

**Reply by pavelzw** [reply]
See the issue in https://github.com/prefix-dev/pixi/issues/891

**Comment by pavelzw** [comment, marked as answer]
Citation is now available 
https://github.com/prefix-dev/pixi/blob/main/CITATION.cff

```bibtex
@software{Arts_pixi,
author = {Arts, Ruben and Zalmstra, Bas and Vollprecht, Wolf and de Jager, Tim},
license = {BSD-3-Clause},
title = {{pixi}},
url = {https://github.com/prefix-dev/pixi/releases/tag/v0.16.1}
}
```

> Arts, R., Zalmstra, B., Vollprecht, W., & de Jager, T. pixi [Computer software]. https://github.com/prefix-dev/pixi/releases/tag/v0.16.1
